src/services/paymentService.js

```diff
@@ -38,24 +36,22 @@ class PaymentService {
         success_url: `${appUrl}/api/v1/payment/success-callback?token=${token}&&paymentId={CHECKOUT_SESSION_ID}`,
         cancel_url: `${appUrl}/api/v1/payment/cancel-callback?token=${token}`,
       });
-      res
-        .status(200)
-        .json({token, paymentId: session.id, url: session.url });
+      res.status(200).json({ token, paymentId: session.id, url: session.url });
     } catch (error) {
-        /* istanbul ignore next */
+      /* istanbul ignore next */
       res.status(500).json({ error });
```

Comment:
here, I think error.message would be much better

Reply:
@jimmymut  I think this code belongs to @teerenzo  worked on payment

Reply:
I realized that after I added the comment